base/uk.ac.stfc.isis.ibex.scriptgenerator/defined_actions/action_loader.py

```diff
@@ -0,0 +1,72 @@
+from py4j.clientserver import ClientServer, JavaParameters, PythonParameters
+from py4j.java_collections import ListConverter
+import inspect
+import argparse
+from action_interface import Action
+import os
+import sys
+
+
+class Config(object):
```

Comment:
I'm not certain that the name Config and ConfigWrapper are completely clear as to their purpose. My understanding of a Config was that it contains multiple actions (which the ConfigWrapper actually does). Maybe the naming should be changed to reflect that, unless there is something I have misunderstood.

Reply:
I've added some documentation for this code here: https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/The-IBEX-Script-Generator#importing-configs-from-user-supplied-actiondefinitions